discussions/no-customizing-classnames.md

```diff
@@ -1,58 +0,0 @@
-# No Customizing `className`'s
```

Comment:
I think the whole no `className` thing still applies.  Customizing the classes through Fela is not the same as customizing it directly.  Its probably worth explaining.